packages/bot-web-ui/src/components/dashboard/joyride-config.tsx

```diff
@@ -507,58 +507,71 @@ export const BOT_BUILDER_MOBILE: TStepMobile[] = [
 export const DBOT_ONBOARDING_MOBILE = [
     {
         header: localize('Get started on DBot'),
-        content: localize('Hi Hit Start for a quick tour to help you get started.'),
+        content: [
+            <Localize
+                key='get-started=mobile'
+                i18n_default_text='Hi Hit <0>Start</0> for a quick tour to help you get started.'
```

Comment:
@rupato-deriv what is the secret tag `<0>` ?

Reply:
This is for custom components on your translation, in this case `<strong>` will replace that tag.